samples/tutorials/Data passing in python components.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
```

Comment:
<p>nit: perhaps prefer not to shadow built-in type name. Can we do <code>list_arg</code>  instead of <code>list</code>  in the signature?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Renamed to list_of_strings</p>